app/app.go

```diff
@@ -168,7 +169,18 @@ func (r *Runner) WithTaskHelpers(t ...taskhelper.Helper) *Runner {
 }
 
 func (r *Runner) WithDefaultHTTPServer(metricNamespace string) *Runner {
```

Comment:
You can call WithDefaultHTTPServerAndPrometheusRegistry from there to avoid repeat yourself